keyboards/horrortroll/nyx/rev2/solder/readme.md

```diff
@@ -0,0 +1,27 @@
+# Nyx (Solder)
+
+![Nyx](https://i.imgur.com/7ccW7wPh.png)
```

Comment:
Dead link.

Reply:
![image](https://github.com/qmk/qmk_firmware/assets/24840279/f2ae8f0f-4b5e-4b09-bce1-f8cd09ae84e1)

That weird, the link still work on my side here. Can you re-check again?

Reply:
Still doesn't work when I click on it or paste it directly, although Camo seems to have no problems displaying it. 🤷